src/Core/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## NOT RELEASED
 
+## 1.4.1
+
+### Fixed
+
+- Make sure passing `@region` to an API operation has effect.
+- Check that both AWS access id and secret exists before using them.
```

Comment:
We should check that change log is update for each PR. (maybe with a not-required test).

This is hard and a time consuming work when releasing new version

Reply:
Do you mean that every PR should include a change log?

I've tried that before, but when you have multiple PR open it will just cause merge conflicts. Ie, you cannot merge two PRs to same component without rebasing one of them first. 

If we want to automate the changelog, we should maybe look info what the official SDK is using.

Reply:
yes this is what I mean,

I'm not sure about automated PR (hard to guess what changed... comments, new constant, deprecation...) 

For conflict, yes, it's a drawback..

It was just a thought about this

Reply:
Let's try to force changelog. Let's see how much problem the conflicts are